README.md

```diff
@@ -585,3 +590,129 @@ point that out explicitly and clearly in the associated patches and Cc
+  `SIGCHLD`/`waitid()` on them or whether waiting for `POLLIN` on
+  them is the only way to get exit notification.
+
+
+* There should be a way to control the process' `comm` field if
+  started via `fexecve()`.
+
+  Right now, when `fexecve()`/`execveat()` is used, the `comm` field
+  (i.e. `/proc/self/comm`) contains a name derived of the numeric fd,
+  which breaks `pc -C …` and various other tools.  In particular when
+  the fd was opened with `O_CLOEXEC`, the number of the fd in the old
+  process is completely meaningless.
+
+  The goal is add a way to tell `fexecve()`/`execveat()` what Name to use.
+
+  Since `comm` is under user control anyway (via `PR_SET_NAME`), it
+  should be safe to also make it somehow configurable at fexecve()
+  time.
+
+  See https://github.com/systemd/systemd/commit/35a926777e124ae8c2ac3cf46f44248b5e147294,
+  https://github.com/systemd/systemd/commit/8939eeae528ef9b9ad2a21995279b76d382d5c81.
+
+  **Usecase:** In systemd we generally would prefer using `fexecve()`
+  to safely and rece-freely invoke processes, but the fact that `comm`
+  is useless after invoking a process that way makes the call
+  unfortunately hard to use for systemd.
+
+* The LSM module API should have the ability to do path-based (not
+  just inode-based) ACL management.
+
+  **Usecase:** This would be useful in BPF-LSM modules such as
+  systemd's `mntfsd` which allows unprivileged file system mounts in
+  some cases, and which would like to restrict ACL handling based on
+  the superblock involved.
+
+* `overlayfs` should permit *immutable* layers, i.e. layers whose
+  non-directory inodes may not be overriden in an upper writable
+  layer.
+
+  **Usecase:** This would be useful when implementing `/etc/` as a
+  stack of overlayfs layers, each shipping configuration for a
+  different facet of the system, with a writable layer on the top for
+  local modifications. In such a scenario it would be useful to allow
+  the user to change any configuration it likes, except of the files
+  and other inodes shipped in the lower layers.
+
+* `overlayfs` should have an `ioctl()`-based API (or similar) for
+  querying information of the backing file systems/block devices
+
+  **Usecase:** In systemd in various areas we automatically find the
+  block device backing the root file system and other file systems
+  (Example: `systemd-gpt-auto-generator` or `bootctl` wull try to find
+  auxiliary file systems of the OS image by looking in the GPT
+  partition table the root file system is located in). While this
+  logic is good enough to find the backing block devices of some more
+  complex storage such as dm-crypt, dm-verity or btrfs, once
+  `overlayfs` is used as backing for the root file system this logic
+  does not work anymore. It would be great if there was an API to
+  simply query `overlayfs` for the superblock information
+  (i.e. `.st_dev`) of the backing layers.
+
+* An *auto-grow* feature in `btrfs` would be excellent.
```

Comment:
Shouldn't this be managed by a user-space utility instead? Have a process that monitors free disk space, and when it's below some configurable amount, e.g. 1G, it resizes the fs?